appveyor.yml

```diff
@@ -26,19 +29,22 @@ install:
   - echo %OMNIBUS_RUBYGEMS%
   - ps: $env:OMNIBUS_BUNDLER=$(findstr bundler omnibus_overrides.rb | %{ $_.split(" ")[3] })
   - echo %OMNIBUS_BUNDLER%
-  - ruby --version
   - gem --version
   - gem update --system %OMNIBUS_RUBYGEMS% || gem update --system %OMNIBUS_RUBYGEMS% || gem update --system %OMNIBUS_RUBYGEMS%
-  - gem --version
-  - bundler --version
-  - SET BUNDLE_WITHOUT=server:docgen:maintenance:pry:travis:integration:ci
   - appveyor DownloadFile http://curl.haxx.se/ca/cacert.pem -FileName C:\cacert.pem
   - set SSL_CERT_FILE=C:\cacert.pem
-  - bundle env
-
-build_script:
+  - SET BUNDLE_WITHOUT=server:docgen:maintenance:pry:travis:integration:ci
+  - bundle config --local path vendor/bundle # use the cache we define above
   - bundle install || bundle install || bundle install
 
+build: off
```

Comment:
what does this do?

Reply:
We're not doing a build anymore. Just a test so this skips that build phase